Code/server/GameServer.cpp

```diff
@@ -747,8 +879,14 @@ void GameServer::HandleAuthenticationRequest(const ConnectionId_t aConnectionId,
     }
 
     // check if the proper server password was supplied.
-    if (acRequest->Token == sPassword.value())
+    if (acRequest->Token == sPassword.value() || (acRequest->Token == sAdminPassword.value() && !sAdminPassword.empty()))
```

Comment:
might be nicer to assign the result of the second expr to a boolean then we can also use the result below

Reply:
Fixed with additional bool above